templates/flavors/vpcless/kustomization.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../default
+
+patches:
+  - target:
+      kind: LinodeVPC
+    patch: |-
+      $patch: delete
+      apiVersion: infrastructure.cluster.x-k8s.io/v1alpha1
+      kind: LinodeVPC
+      metadata:
+        name: ${VPC_NAME:=${CLUSTER_NAME}}
+  - target:
+      group: infrastructure.cluster.x-k8s.io
+      version: v1alpha1
+      kind: LinodeCluster
+    patch: |-
+      - op: remove
+        path: /spec/vpcRef
+  - target:
+      group: controlplane.cluster.x-k8s.io
+      version: v1beta1
+      kind: KubeadmControlPlane
+    patch: |-
+      - op: remove
+        path: /spec/kubeadmConfigSpec/initConfiguration/skipPhases
+  - target:
+      group: cluster.x-k8s.io
+      version: v1beta1
+      kind: Cluster
+    patch: |-
+      apiVersion: cluster.x-k8s.io/v1beta1
+      kind: Cluster
+      metadata:
+        name: ${CLUSTER_NAME}
+        labels:
+            cni: cilium
+            vxlan: "true"
+            ccm: linode
+            vpcless: "true"
```

Comment:
For some reason I don't see the vpcless label when using dual-stack, we might want to add that in the dual-stack kustomization with a comment that it's there until ipv6 support works for VPCs?

Reply:
Did you do `make local-release` before provisioning cluster? I am able to see the right labels for both vpcless and dual-stack clusters.
```
k get cluster vpcless -o yaml | yq .metadata.labels
ccm: linode
cluster: vpcless
cni: cilium
crs: vpcless-crs
csi: linode
vpcless: "true"
vxlan: "true"
```

```
k get cluster dual-stack -o yaml | yq .metadata.labels
ccm: linode
cluster: dual-stack
cni: cilium
crs: dual-stack-crs
csi: linode
ipv6: "true"
vpcless: "true"
vxlan: "true"
```

Reply:
Looks like my clusterctl config file needed to be updated since the path changed again for the local release.

Reply:
Hmm, but the above output means CNI won't get installed in dual-stack cluster. We'll need to revert the tags check. 😕